src/engine/instrument.cpp

```diff
@@ -70,6 +70,7 @@ void DivInstrument::putInsData(SafeWriter* w) {
 
   for (int j=0; j<4; j++) {
     DivInstrumentFM::Operator& op=fm.op[j];
+    w->writeC(op.enable?1:0);
```

Comment:
Please put this at the end of file - I don't want to break the format like this...